libs/sdk-core/src/swap.rs

```diff
@@ -400,9 +400,9 @@ impl BTCReceiveSwap {
                     amount_sats: swap_info.confirmed_sats,
                     description: String::from("Bitcoin Transfer"),
                     preimage: Some(swap_info.preimage),
-                    opening_fee_params: None,
+                    opening_fee_params: swap_info.channel_opening_fees,
                     use_description_hash: Some(false),
-                    expiry: None,
+                    expiry: Some(SWAP_PAYMENT_FEE_EXPIRY),
```

Comment:
Do you see any issue with that? I think it is better and ensures that the invoice expiry is always after the expiration of the fees. Since we pass the fees explicitly here it doesn't affect the fees selection.

Reply:
If we want to minimize the sender's insight in the fact that the recipient is using an LSP, we should make the invoice expire before the fee params expire. Otherwise the sender could get unknown_next_peer, and the sender will probably not be able to pay a subsequent invoice the way the LSP works now. Once we get a new scid for every channel open that won't be much of an issue, but in any case unknown_next_peer should be avoided as muchc as possible in my opinion.

Reply:
As discussed internally we will leave it as is now. The only payer currently at this use case is our swapper.

Reply:
I can still see an issue here in the following use case:
1. User created a swap address
2. Wait for two days before sending onchain funds.
3. Then send on chain funds and not able to create invoice.

Reply:
After our internal discussion. Moving back to the non-strict approach where we are not creating any coupling between the expiry and the lsp fee validity.